builder/dockerfile/constants.go

```diff
@@ -0,0 +1,13 @@
+package dockerfile
+
+// Build metrics prometheus messages
+const (
```

Comment:
Maybe a comment explaining that values must be initialized in builder/dockerfile/metrics.go... it may be good to put these in metrics.go even.

We should explain how to add new metrics to this package so we don't end up with random stuff added.